File: day014/day014.cs
using System;
class Day014 {
  static void Main() {
    int x, y;
    
    Console.Write("Bir sayı giriniz: ");
    x = Convert.ToInt32(Console.ReadLine());

    Console.Write("Bir sayı giriniz: ");
    y = Convert.ToInt32(Console.ReadLine());
    
    if (x % y == 0 || y % x == 0)
        Console.WriteLine("Girdiğiniz sayılar birbirlerinin katlarıdır.");
   
    else
        Console.WriteLine("Girdiğiniz sayılar birbirlerinin katları değildir.");
  }
}

File: day014/day014.py
x = int(input("Bir sayı giriniz: "))
y = int(input("Bir sayı giriniz: "))  
    
if x % y == 0 or y % x == 0:
    print("Girdiğiniz sayılar birbirlerinin katlarıdır.")
else:
    print("Girdiğiniz sayılar birbirlerinin katları değildir.")
